tools/pepsi/src/analyze.rs

```diff
@@ -40,6 +40,7 @@ pub async fn analyze(arguments: Arguments, repository: &Repository) -> Result<()
             let file_path = repository
                 .find_latest_file(&format!("target/**/{crate_name}-*/**/{file_name}"))
                 .wrap_err("failed find latest build script output")?;
+            println!("{}", file_path.display());
```

Comment:
Pepsi analzye is useful because it shows you the contents of generated files.
That's nice, but it doesn't give you a full LSP inside the pager.
For more detailed analysis, I found it useful to just open the file in my editor.
Before this change there was no way to get the actual path of the file that was displayed in the pager.

I think this should stay.

Reply:
We should rather investigate why `bat` does not show it properly. It should be able to and I think I looked into it shortly but had other things to do. Maybe you could research again, why it does not show it and how to enable it. We should aim to drop the manual `println`.

Reply:
It seems the bat library never adds the `HeaderFilename` style component regardless of what options you set.
[Here](https://docs.rs/bat/0.22.1/src/bat/pretty_printer.rs.html#280) it sets the header component but not header filename.

Reply:
See https://github.com/sharkdp/bat/issues/2378 and https://github.com/sharkdp/bat/pull/2406. I'm not sure what we do in the meantime. 🤔

Reply:
I suggest leaving it as is for now. Will revisit once the changes in bat are merged.